openapi3/schema.go

```diff
@@ -1687,6 +1714,12 @@ func (err *SchemaError) JSONPointer() []string {
 }
 
 func (err *SchemaError) Error() string {
+	if err.customizeMessageError != nil {
+		if msg := err.customizeMessageError(err); msg != "" {
```

Comment:
The documentation of this option should mention that returning an empty string falls back to the upstream `Error()` impl.

Reply:
added